# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR pyLoad Team
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: pyLoad 0.4\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2010-10-07 22:49+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: pyLoadCore.py:163
msgid "Received Quit signal"
msgstr ""

#: pyLoadCore.py:205
#, python-format
msgid "Failed changing group: %s"
msgstr ""

#: pyLoadCore.py:214
#, python-format
msgid "Failed changing user: %s"
msgstr ""

#: pyLoadCore.py:216
msgid "folder for logs"
msgstr ""

#: pyLoadCore.py:227
#, python-format
msgid "Using home directory: %s"
msgstr ""

#: pyLoadCore.py:231
msgid "pycrypto to decode container files"
msgstr ""

#: pyLoadCore.py:232
msgid "Python Image Libary (PIL) for captcha reading"
msgstr ""

#: pyLoadCore.py:234
msgid "Django for webinterface"
msgstr ""

#: pyLoadCore.py:235
msgid "folder for temporary files"
msgstr ""

#: pyLoadCore.py:236
msgid "tesseract for captcha reading"
msgstr ""

#: pyLoadCore.py:240
msgid "folder for downloads"
msgstr ""

#: pyLoadCore.py:241
msgid "file for links"
msgstr ""

#: pyLoadCore.py:244
msgid "OpenSSL for secure connection"
msgstr ""

#: pyLoadCore.py:268
#, python-format
msgid "Downloadtime: %s"
msgstr ""

#: pyLoadCore.py:279
#, python-format
msgid "Free space: %sGB"
msgstr ""

#: pyLoadCore.py:281
#, python-format
msgid "Free space: %sMB"
msgstr ""

#: pyLoadCore.py:302
msgid "links.txt could not be cleared"
msgstr ""

#: pyLoadCore.py:319
msgid "restarting pyLoad"
msgstr ""

#: pyLoadCore.py:323
msgid "pyLoad quits"
msgstr ""

#: pyLoadCore.py:344
msgid "Secure XMLRPC Server Started"
msgstr ""

#: pyLoadCore.py:346
msgid "SSL Certificates not found, fallback to auth XMLRPC server"
msgstr ""

#: pyLoadCore.py:348 pyLoadCore.py:351
msgid "Auth XMLRPC Server Started"
msgstr ""

#: pyLoadCore.py:357
#, python-format
msgid "Failed starting XMLRPC server CLI and GUI will not be available: %s"
msgstr ""

#: pyLoadCore.py:401
#, python-format
msgid "Install %s"
msgstr ""

#: pyLoadCore.py:436
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr ""

#: pyLoadCore.py:438
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr ""

#: pyLoadCore.py:463
msgid "shutting down..."
msgstr ""

#: pyLoadCore.py:478
msgid "error while shutting down"
msgstr ""

#: pyLoadCore.py:611
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr ""

#: pyLoadCore.py:809
msgid "killed pyLoad from Terminal"
msgstr ""

#: module/ThreadManager.py:94
#, python-format
msgid "Reconnect Failed: %s"
msgstr ""

#: module/ThreadManager.py:115
msgid "Reconnect script not found!"
msgstr ""

#: module/ThreadManager.py:121
msgid "Starting reconnect"
msgstr ""

#: module/ThreadManager.py:130
#, python-format
msgid "Old IP: %s"
msgstr ""

#: module/ThreadManager.py:135
msgid "Failed executing reconnect script!"
msgstr ""

#: module/ThreadManager.py:151
#, python-format
msgid "Reconnected, new IP: %s"
msgstr ""

#: module/HookManager.py:50
#, python-format
msgid "Error executing hooks: %s"
msgstr ""

#: module/HookManager.py:64
#, python-format
msgid "%(name)s loaded, activated %(value)s"
msgstr ""

#: module/HookManager.py:66
#, python-format
msgid "Failed activating %(name)s"
msgstr ""

#: module/FileDatabase.py:84
msgid "finished"
msgstr ""

#: module/FileDatabase.py:84
msgid "offline"
msgstr ""

#: module/FileDatabase.py:84
msgid "online"
msgstr ""

#: module/FileDatabase.py:84
msgid "queued"
msgstr ""

#: module/FileDatabase.py:84
msgid "checking"
msgstr ""

#: module/FileDatabase.py:84
msgid "waiting"
msgstr ""

#: module/FileDatabase.py:84
msgid "reconnected"
msgstr ""

#: module/FileDatabase.py:84
msgid "starting"
msgstr ""

#: module/FileDatabase.py:84
msgid "failed"
msgstr ""

#: module/FileDatabase.py:84
msgid "aborted"
msgstr ""

#: module/FileDatabase.py:84
msgid "decrypting"
msgstr ""

#: module/FileDatabase.py:84
msgid "custom"
msgstr ""

#: module/FileDatabase.py:84
msgid "downloading"
msgstr ""

#: module/FileDatabase.py:84
msgid "processing"
msgstr ""

#: module/FileDatabase.py:84
msgid "unknown"
msgstr ""

#: module/FileDatabase.py:499 module/plugins/hooks/IRCInterface.py:68
#: module/plugins/hooks/XMPPInterface.py:75
#, python-format
msgid "Package finished: %s"
msgstr ""

#: module/FileDatabase.py:610
msgid "Filedatabase was deleted due to incompatible version."
msgstr ""

#: module/PluginManager.py:79
msgid "Crypter"
msgstr ""

#: module/PluginManager.py:80
msgid "Container"
msgstr ""

#: module/PluginManager.py:81
msgid "Hoster"
msgstr ""

#: module/PluginManager.py:83
msgid "Captcha"
msgstr ""

#: module/PluginManager.py:84
msgid "Account"
msgstr ""

#: module/PluginManager.py:85
msgid "Hook"
msgstr ""

#: module/PluginManager.py:87
msgid "created index of plugins"
msgstr ""

#: module/PluginManager.py:166
#, python-format
msgid "%s has invalid pattern."
msgstr ""

#: module/PluginManager.py:293
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr ""

#: module/PluginManager.py:294
msgid "You should fix dependicies or deactivate load on startup."
msgstr ""

#: module/AccountManager.py:85
msgid "Account settings deleted, due to new config format."
msgstr ""

#: module/PluginThread.py:150
#, python-format
msgid "Download skipped: %(name)s @ %(plugin)s"
msgstr ""

#: module/PluginThread.py:153
msgid "File already exists."
msgstr ""

#: module/PluginThread.py:156
#, python-format
msgid "Download starts: %s"
msgstr ""

#: module/PluginThread.py:164 module/PluginThread.py:319
#, python-format
msgid "Plugin %s is missing a function."
msgstr ""

#: module/PluginThread.py:173 module/PluginThread.py:226
#, python-format
msgid "Download aborted: %s"
msgstr ""

#: module/PluginThread.py:192
#, python-format
msgid "Download restarted: %s"
msgstr ""

#: module/PluginThread.py:202 module/PluginThread.py:328
#, python-format
msgid "Download is offline: %s"
msgstr ""

#: module/PluginThread.py:205 module/PluginThread.py:251
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr ""

#: module/PluginThread.py:218
msgid ""
"Couldn't connect to host or connection resetted waiting 1 minute and retry."
msgstr ""

#: module/PluginThread.py:268
#, python-format
msgid "Download finished: %s"
msgstr ""

#: module/PluginThread.py:314
#, python-format
msgid "Decrypting starts: %s"
msgstr ""

#: module/PluginThread.py:331 module/PluginThread.py:344
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr ""

#: module/PluginThread.py:338
#, python-format
msgid "Retrying %s"
msgstr ""

#: module/web/ServerThread.py:45
msgid "Database for Webinterface does not exitst, it will not be available."
msgstr ""

#: module/web/ServerThread.py:46
msgid "Please run: python pyLoadCore.py -s"
msgstr ""

#: module/web/ServerThread.py:47
msgid ""
"Go through the setup and create a database and add an user to gain access."
msgstr ""

#: module/web/ServerThread.py:88
msgid "SSL certificates not found."
msgstr ""

#: module/web/ServerThread.py:98
#, python-format
msgid ""
"Can't use %(server)s, either python-flup or %(server)s is not installed!"
msgstr ""

#: module/web/ServerThread.py:132
#, python-format
msgid "Starting nginx Webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:169
#, python-format
msgid "Starting lighttpd Webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:179
#, python-format
msgid "Starting django builtin Webserver: %(host)s:%(port)d"
msgstr ""

#: module/plugins/Crypter.py:58
#, python-format
msgid "Parsed package %(name)s with %(len)d links"
msgstr ""

#: module/plugins/Plugin.py:252
msgid ""
"Pil and tesseract not installed and no Client connected for captcha "
"decrypting"
msgstr ""

#: module/plugins/Plugin.py:254
msgid "No Client connected for captcha decrypting"
msgstr ""

#: module/plugins/Plugin.py:296 module/plugins/Plugin.py:319
#, python-format
msgid "Setting User and Group failed: %s"
msgstr ""

#: module/plugins/Container.py:69
msgid "File not exists."
msgstr ""

#: module/plugins/Account.py:50 module/plugins/Account.py:54
#, python-format
msgid "Could not login with %(plugin)s account %(user)s | %(msg)s"
msgstr ""

#: module/plugins/Account.py:50
msgid "Wrong Password"
msgstr ""

#: module/plugins/Account.py:148
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr ""

#: module/plugins/Account.py:185
#, python-format
msgid "%(plugin)s Account %(user)s has not enough traffic"
msgstr ""

#: module/plugins/Account.py:190
#, python-format
msgid "%(plugin)s Account %(user)s is expired"
msgstr ""

#: module/plugins/crypter/SerienjunkiesOrg.py:103
msgid "Downloadlimit reached"
msgstr ""

#: module/plugins/crypter/CryptItCom.py:40
msgid "File not found"
msgstr ""

#: module/plugins/hooks/ClickAndLoad.py:73
msgid "Click'N'Load: Port 9666 already in use"
msgstr ""

#: module/plugins/hooks/UnRar.py:118
#, python-format
msgid "starting Unrar of %s"
msgstr ""

#: module/plugins/hooks/UnRar.py:134 module/plugins/hooks/UnRar.py:164
#, python-format
msgid "Unrar of %s failed (wrong password)"
msgstr ""

#: module/plugins/hooks/UnRar.py:140
#, python-format
msgid "Unrar of %s failed (missing volume)"
msgstr ""

#: module/plugins/hooks/UnRar.py:144 module/plugins/hooks/UnRar.py:161
#, python-format
msgid "Unrar of %s ok"
msgstr ""

#: module/plugins/hooks/UnRar.py:149 module/plugins/hooks/UnRar.py:157
#, python-format
msgid "Unrar of %s failed"
msgstr ""

#: module/plugins/hooks/UnRar.py:152
#, python-format
msgid ""
"Your ram amount of %s MB seems not sufficient to unrar this file. You can "
"deactivate this warning and risk instability"
msgstr ""

#: module/plugins/hooks/IRCInterface.py:75
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr ""

#: module/plugins/hooks/XMPPInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr ""

#: module/plugins/hooks/XMPPInterface.py:114
#, python-format
msgid "pyLoad XMPP: Message from %s received."
msgstr ""

#: module/plugins/hooks/XMPPInterface.py:115
#, python-format
msgid "pyLoad XMPP: Body: %s"
msgstr ""

#: module/plugins/hooks/XMPPInterface.py:169
#, python-format
msgid "pyLoad XMPP: Send message to %s"
msgstr ""

#: module/plugins/hooks/HotFolder.py:78
#, python-format
msgid "Added %s from HotFolder"
msgstr ""

#: module/plugins/hooks/ExternalScripts.py:47
#: module/plugins/hooks/ExternalScripts.py:48
msgid "folders for scripts"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:43
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:45
msgid "No plugin updates available"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:53
msgid "No Updates for pyLoad"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:56
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:57
msgid "***  Get it here: http://pyload.org/download  ***"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:60
#: module/plugins/hooks/UpdateManager.py:70
msgid "Not able to connect server for updates"
msgstr ""

#: module/plugins/hooks/UpdateManager.py:95
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr ""

#: module/plugins/hoster/MegauploadCom.py:83
#, python-format
msgid "Megaupload: waiting %d minues"
msgstr ""

#: module/plugins/hoster/MegauploadCom.py:108
#, python-format
msgid ""
"%s: Megaupload is currently blocking your IP. Try again later, manually."
msgstr ""

#: module/plugins/hoster/MegauploadCom.py:175
msgid ""
"Looks like the file is still not available. Retry downloading later, "
"manually."
msgstr ""

#: module/plugins/hoster/UploadedTo.py:56
#: module/plugins/hoster/RapidshareCom.py:153
#, python-format
msgid "%(name)s: Use Premium Account (%(left)sGB left)"
msgstr ""

#: module/plugins/hoster/UploadedTo.py:58
#: module/plugins/hoster/RapidshareCom.py:155
#, python-format
msgid "%s: Not enough traffic left"
msgstr ""

#: module/plugins/hoster/UploadedTo.py:61
msgid "Traffic exceeded"
msgstr ""

#: module/plugins/hoster/RapidshareCom.py:98
msgid "Rapidshare: Traffic Share (direct download)"
msgstr ""

#: module/plugins/hoster/RapidshareCom.py:139
#: module/plugins/hoster/RapidshareCom.py:213
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr ""

#: module/plugins/hoster/RapidshareCom.py:144
msgid "Invalid Auth Code, download will be restarted"
msgstr ""

#: module/plugins/hoster/RapidshareCom.py:217
msgid "RapidShareCom: No free slots"
msgstr ""

#: module/plugins/hoster/NetloadIn.py:166
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr ""

#: module/plugins/hoster/NetloadIn.py:191
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr ""

#: module/plugins/hoster/NetloadIn.py:230
msgid "Downloaded File was empty"
msgstr ""
